### PR TITLE
Updating to automatic and multilanguage package

### PR DESCRIPTION
**Pull request by adgellida**
See the changes and apport changes if it's necessary

### CODE REVIEW

**Comment by ferventcoder** [comment]
Thanks, I will evaluate this and get back to you on it.

**Comment by ferventcoder** [comment]
Still holding on this guy until I can test it properly.

File: adobereader/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,8 +1,42 @@
-$silentArgs = '/sPB /msi /norestart /quiet ALLUSERS=1 EULA_ACCEPT=YES'
-$downloadUrl = '{{DownloadUrl}}'
+$packageName = 'adobereader'
+$installerType = 'EXE'
+$silentArgs = '/sAll /msi /norestart /quiet ALLUSERS=1 EULA_ACCEPT=YES'
+#$url = 'http://ardownload.adobe.com/pub/adobe/reader/win/{mainversion}.x/{version}/en_US/AdbeRdr{version:replace:.:}_en_US.exe'
```
Comment (ferventcoder) [inline adobereader/tools/chocolateyInstall.ps1:4]: This line is incorrect. {version:replace:.:} is a function of ketarin, not of powershell.

Comment (adgellida) [inline adobereader/tools/chocolateyInstall.ps1:4]: ok, adobereader isn't multi yet? u can do it manual but multi if not?

2014-02-09 21:33 GMT+01:00 Rob Reynolds notifications@github.com:

Comment (ferventcoder) [inline adobereader/tools/chocolateyInstall.ps1:4]: Note that you are attempting to call a KETARIN function in PowerShell which does not know `{mainversion}` `{version}` or `{version.replace:.:}`. This looks to be a copy/paste error. (Oh wait) you have this commented.

Nevermind. Ignore this. :)

**Comment by ferventcoder** [comment]
I'm pulling in the gist of this into https://github.com/ferventcoder/chocolatey-packages

**Comment by adgellida** [comment]
Hurray! But now I'm on the Open Source wave with your Sumatra package. For
all the other people it will be find, so delete:

<goog_1658066316>
https://chocolatey.org/packages/adobereader-de
https://chocolatey.org/packages/adobereader-jp
https://chocolatey.org/packages/simnetsa-adobereader-fr
https://chocolatey.org/packages/BR.AdobeReaderFR

Ok?

2014-09-24 15:11 GMT+02:00 Rob Reynolds notifications@github.com:

**Pull request closed without merging**